subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -819,6 +822,18 @@ l2cap_br_conn_security(struct bt_l2cap_chan *chan, const uint16_t psm)
 		break;
 	}
 
+	if (chan->conn->sec_level < br_chan->required_sec_level &&
```

Comment:
@zhongzhijie1 could you please check if it was necessary to disconnect ACL connection？